config/locales/support_interface/support_interface.yml

```
@@ -191,12 +191,19 @@ en:
               too_many_words: Tell us about the candidate’s subject knowledge in %{count} words or fewer
             audit_comment:
               blank: You must provide an audit comment
-        support_interface/application_forms/edit_gcse_form:
+        support_interface/application_forms/edit_gcse_award_year_form:
           attributes:
             award_year:
               blank: Award year cannot be blank
+              wrong_length: Enter a valid award year (for exmaple, 2001)
             audit_comment:
-              blank: You must provide an audit comment
+              blank: Enter a Zendesk ticket URL
+        support_interface/application_forms/edit_gcse_grade_form:
+          attributes:
+            grade:
+              blank: Grade cannot be blank
```

I think the standard pattern for "blank" error messages is "Enter [xxx]".

```suggestion
              blank: Enter the grade
```